test/org/traccar/protocol/OwnTracksProtocolDecoderTest.java

```diff
@@ -0,0 +1,26 @@
+package org.traccar.protocol;
+
+import org.junit.Test;
+import org.traccar.ProtocolTest;
+
+public class OwnTracksProtocolDecoderTest extends ProtocolTest {
+
+    @Test
+    public void testDecode() throws Exception {
+
+        OwnTracksProtocolDecoder decoder = new OwnTracksProtocolDecoder(new OwnTracksProtocol());
+
+/*
+ * I don'w know how to get JSON into that ...
```

Comment:
I would gladly provide unit tests, but I get the following error when providing the JSON as shown in the code:

```
Tests in error:
  testDecode(org.traccar.protocol.OwnTracksProtocolDecoderTest): Invalid token=EOF at (line no=1, column no=0, offset=-1). Expected tokens are: [CURLYOPEN, SQUAREOPEN]
```

can you please help me understand that? Expected CURLYOPEN I assume is a `{` which is there ...